libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -616,16 +616,76 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
         border-bottom-right-radius: 0;
         border-top-right-radius: 0;
 
+        &:not(.disabled):not(.is-active)::before {
```

Comment:
`:not` is not supported by IE
https://caniuse.com/?search=not

Reply:
Please resolve all the checkbox in PR Checklist if possible

Reply:
I have changed it thanks